src/test/unit/function-info.test.js

```diff
@@ -0,0 +1,22 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { stripFunctionArguments } from '../../profile-logic/function-info';
+
+describe('get-function-name', function () {
+  it('should strip the function arguments', function () {
+    expect(stripFunctionArguments('ns::fn()')).toEqual('ns::fn');
+    expect(stripFunctionArguments('ns::fn(bool (*)(JS::Handle<JSObject*>))')).toEqual('ns::fn');
+    expect(stripFunctionArguments('Bar* ns::fn<(Foo)0>(void const*, unsigned int)')).toEqual('Bar* ns::fn<(Foo)0>');
```

Comment:
That's a good question.

For example I have this from an example:
`js::detail::HashTable<js::AtomStateEntry const, js::HashSet<js::AtomStateEntry, js::AtomHasher, js::SystemAllocPolicy>::SetOps, js::SystemAllocPolicy>::lookupForAdd(js::AtomHasher::Lookup const&) const`

I _think_ "copy function name" should copy `js::detail::HashTable::lookupForAdd` and get rid of the template information.

What do you think @mstange ?